navbar-service.js

```diff
@@ -28,6 +32,23 @@ function factory(config) {
   // exposes menus for configuration
   service.menus = {};
 
+  service.site = {
+    title: config.data.siteTitle,
+    brand: config.data.style.brand,
+    url: '/'
+  };
```

Comment:
That said, I think what we **really** want, is a solution that would set the navbar appropriately for
`/demo/[tenant]/foo` and `/demo/[tenant]/bar` etc. without having to copy boilerplate to every route.  This is to ensure that all `[tenant]` routes are properly branded.